README.md

```diff
@@ -36,7 +36,7 @@ What `fxl` attempts to do differently to [docjure](https://github.com/mjul/docju
 
 ## Map Representation of Cells
 
-A `fxl` cell is represented by a map that tells us its value, location and style. For instance:
+A `fxl` cell is represented by a map that tells us its value, location and style. Style is optional. For instance:
```

Comment:
Let's keep the styles in the examples. Then before the end of the section, say that the style is optional, so that "this cell is equal to that cell"?

Reply:
@anthony-khong in both the README and the `example/write_to_plain_excel.clj`?

Am thinking maybe keep style in the readme examples and say that style is optional such that it will be the same as the code in `example/write_to_plain_excel.clj` ?

Reply:
On second thought that's probably just confusing. Will make the edits